bigbluebutton-html5/imports/ui/components/nav-bar/nav-bar-graphql/recording-indicator/component.tsx

```diff
@@ -147,7 +147,7 @@ const RecordingIndicator: React.FC<RecordingIndicatorProps> = ({
 
   const recordingIndicatorIcon = useMemo(() => (
     <Styled.RecordingIndicatorIcon
-      titleMargin={!isPhone && recording}
+      titleMargin={!isPhone}
```

Comment:
margin needs to be applied when the device is a phone and recording is active

![Screenshot from 2024-07-08 09-12-26](https://github.com/bigbluebutton/bigbluebutton/assets/3728706/6252947d-4706-4bec-97a9-1bbdb8f357fa)

```suggestion
      titleMargin={!isPhone || recording}
```